pkg/provisioner/ironic/validatemanagementaccess_test.go

```diff
@@ -107,6 +107,44 @@ func TestValidateManagementAccessCreateNodeNoImage(t *testing.T) {
 	assert.Equal(t, createdNode.DeployInterface, "")
 }
 
+func TestValidateManagementAccessCreateNodeNewInspection(t *testing.T) {
```

Comment:
Do we need a testcase to check if the backward compatibility is working with `inspector`  interface as well? Or is it already tested in other cases? Sorry I am a bit unfamiliar with unit tests in this part of the code